.github/workflows/zeyple.yml

```diff
@@ -0,0 +1,33 @@
+name: Zeyple test and deb package build
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v1
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 2.7
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y ruby ruby-dev rubygems build-essential gnupg python-gpg libgpgme-dev swig
```

Comment:
What do we need the `swig` package for?

Reply:
It seems that the `gpg` Python dependency needs it: https://github.com/DrauzJu/zeyple/commit/1344e64f7d5a4815e7ff61b71be6b8109293d45b/checks?check_suite_id=262393072#step:4:328